chain/precompile/router/router.go

```diff
@@ -39,3 +39,11 @@ func (c *Contract) Send(
 	_, err := c.rtr.Send(ctx, namespace, pCtx.MsgSender().String(), messageID, message)
 	return nil, err
 }
```

Comment:
We should probably make it clearer here `sendMessage` sounds about right

Reply:
not entirely sure whats being said here

Reply:
yeah

Reply:
updooted [refactor: send -> SendMessage](https://github.com/Argus-Labs/world-engine/pull/186/commits/a2c7e08701ffed26bbfec8453262e40655a3b9df)